Formula/vim.rb

```diff
@@ -57,12 +54,12 @@ def install
                           "--mandir=#{man}",
                           "--enable-multibyte",
                           "--with-tlib=ncurses",
-                          "--with-compiledby=Homebrew",
+                          "--with-compiledby=#{tap.user}",
                           "--enable-cscope",
                           "--enable-terminal",
                           "--enable-perlinterp",
                           "--enable-rubyinterp",
-                          "--enable-python3interp",
+                          "--enable-python3interp#{"=dynamic" if OS.linux?}",
```

Comment:
Why not dynamic on macOS?

Reply:
Already dynamic, setting `=dynamic` on macOS breaks `brew test vim`